task18/css/reset.css

```diff
@@ -0,0 +1,108 @@
+html{
+	color:#000;
+	background:#FFF;
+}
+
```

Comment:
write  reset.css reference

Reply:
Thank you for your comment.
I learned a lot because I was not aware that I had to write it.
[f74da7d](https://github.com/seeeen044/JavaScript_handson/pull/19/commits/f74da7d4950575b0b2d7099a0faeb20b696f86de)